Sources/IRGen/Function/IRStatement.swift

```diff
@@ -282,3 +284,12 @@ struct IREmitStatement {
     return IRFunctionCall(functionCall: emitStatement.functionCall).rendered(functionContext: functionContext)
   }
 }
+
+struct IRDoCatchStatement {
+  var doCatchStatement: DoCatchStatement
+
+  func rendered(functionContext: FunctionContext) -> String {
+    // TODO(ethan) implement exception handling codegen
```

Comment:
Please create a ticket for this if there isn't one already and remove the comment. If you want to make a note of the source line then put it on the relevant ticket.

Additionally I think it would be more productive to call `fatalError(...)` for anything that is not yet implemented to prevent us from calling it accidentally. You can still run AST tests if you don't invoke the codegen and this ticket was only to parse (if I understand correctly)

Reply:
Should we create issues and then link these TODOs to those tickets?

Reply:
I think it's sufficient to simply have the issues, perhaps link to the relevant code from the issue.

Reply:
Noted in https://github.com/flintrocks/flint/issues/87